### PR TITLE
Forget password/Change password pages

### PR DESCRIPTION
**Pull request by hgvo9**
This pull request will contains two new pages for Forgetting his password and changing his password and the profile page to edit the user's informations. It will also contains few commits from the branch units-tests which implements units tests.

Main (upcoming) changes from this pull requests : 

- Implementing deep-linking for when the user clicks on a link such as the one to reset your password that you receive in your mail it redirects in the mobile app to change your password for Android (done) and iOS (not done yet 300521)
- Units tests and SDK of appcenter implemented
- Change/Forget password pages done
- Profile page to edit user information done

iOS require a subscription to apple developer program in order to implement deep-linking with universal

### CODE REVIEW

File: src/utils/utils.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+export const extractTokenAndUserId = (url: string) => {
```
Comment (hgvo9) [inline src/utils/utils.ts:1]: @CptHash @Trushy @louischristner @Iboakg5745 I would like you guys to review this little piece of code. I've made this function to extract the userId and the token from the url sent to my mailbox in order to change my password. But I'm pretty sure that I've done it in a bad way as you can see, I didn't figure out the exact regex to extract the token and the userId at once from the url, that's why I would to have you guys review my code, thank you.

**Review by hgvo9 — COMMENTED** [review]
Comment to be deleted

File: src/utils/utils.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    // userid = 594989
+    url = url.substring(url.indexOf("/") + 1, url.length);
+    // Url = token/qwe23909sdfs0934
+    url = url.substring(url.indexOf("/") + 1, url.length);
+    // Url = qwe23909sdfs0934
+    token = url;
+
+    return {userId: userId, token: token};
+
+}
```
[the remark this answers is not in the document]
Comment (hgvo9) [reply src/utils/utils.ts:25]: Trushy has sent me this regex : https:\/\/front\.safely-app\.fr\/reset\/(.*)\/token\/(.*) , and it works ty

**Review by louischristner — APPROVED** [review]
It looks good to me, great job !

**Pull request merged**